### ISSUE TITLE
Incorrect model call

### ISSUE THREAD

**Issue by uladzislau-varabei** [mseitzer/pytorch-fid#118]
Hello,

Thanks for the amazing work porting the official TensorFlow implementation into PyTorch. I'm using your repo for FID evaluation when training my GAN, however, after working with it for a while I noticed that one very important thing is missing. You don't use model in `eval` mode. With my model results are totally different: after 100k images I get score around 20, while with eval mode enabled it increases to a huge value of more than 400(!). For a model that is fully trained the results are also very different: around 2 with default mode and more than 9 in eval mode. I would suggest that you add this small fix that changes results significantly.

Thanks!

**Comment by uladzislau-varabei** [comment]
@mseitzer I think you could be interested in it. I might submit a PR for you in a while if you need it
